Info about webhookEnabled for the build environment (#464)
equinor/radix-public-site: public-site/docs/radix-config/index.md

          VAR3: val3 #env-var VAR3 only exists in the "prod" custom sub-pipeline, in addition to common VAR1 and CONNECTION_STRING 
```

### `webhookEnabled`

```yaml
spec:
  environments:
    - name: dev
      build:
        from: master
        webhookEnabled: false
```

> `webhookEnabled` - (optional, default `true`) enables or disables the webhook for the build environment. If set to `false`, commit on GitHub repo will not trigger trough GitHub webhook a build-deploy pipeline job for this environment.

Reviewer: ```suggestion
`webhookEnabled` - (optional, default `true`) controls whether Github webhook shall create `build-deploy` pipeline jobs on commits to the branch.  
If multiple environments are mapped to the same branch, a `build-deploy` pipeline job is created when `webhookEnabled` is true for at least one environment. Environments with `webhookEnabled` set to false will excluded by the pipeline job.
```
Added info about logic when multiple environments are mapped to the same branch

> `webhookEnabled` - (optional, default `true`) enables or disables the webhook for the build environment. If set to `false`, commit on GitHub repo will not trigger trough GitHub webhook a build-deploy pipeline job for this environment.

Reviewer: ```suggestion
`webhookEnabled` - (optional, default `true`) controls whether the environment is built and deployed by `build-deploy` pipeline jobs triggered from Github webhook.
```
